src/common/Assert.h

```diff
@@ -31,6 +31,12 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef COMMON_ASSERT_H_
 #define COMMON_ASSERT_H_
 
+#include <utility>
```

Comment:
Our guidance on the common headers for now is they are not actually expected to be self-contained. What you are supposed to to is always include `Common.h` first in every .cpp file.

Reply:
I've tried to reproduce the bug to try to find an alternative solution. But somehow, the bug doesn't trigger anymore. Do you have suggestions on how to reproduce again?

In the meantime, I removed the commit so be able to merge this faster. Will merge tomorrow most likely